prism-languages.php

```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
```

Comment:
:no_entry_sign: **Error**: There must be exactly one blank line after the file comment (*Squiz.Commenting.FileComment.SpacingAfterComment*).